src/tests/health_test.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/Real-Dev-Squad/gopher-cloud-service/src/routes"
+	"github.com/Real-Dev-Squad/gopher-cloud-service/src/tests/fixtures"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHealthRoute(t *testing.T) {
+	os.Setenv("env", "test")
+	env := os.Getenv("env")
+	version := strconv.Itoa(1)
+	ver := "/v" + version
+	router := routes.SetupRouter(env, version)

```

Comment:
I think this should not be handlers, There should be a setup function for it and for above also

Reply:
it will be managed when viper comes into action in future PRs

Reply:
So on every route test, SetupRouter will be called?

Reply:
yes.

Reply:
https://gin-gonic.com/docs/testing/

Reply:
Will take care of it in future PRs